docs/trulens/tracking/logging/where_to_log/log_in_snowflake.md

```diff
@@ -41,7 +41,7 @@ Connecting TruLens to a Snowflake database for logging traces and evaluations on
     )
     ```
 
-Once you've instantiated the `Tru` object with your Snowflake connection, all _TruLens_ traces and evaluations will logged to Snowflake.
+Once you've instantiated the `TruSession` object with your Snowflake connection, all _TruLens_ traces and evaluations will logged to Snowflake.
 
 ## Connect TruLens to the Snowflake database using an engine
 
```

Comment:
can you update later in this same docs page with the latest on using Snowflake Connector, and updating Tru to TruSession?